scripts/translations_util.py

```diff
@@ -1,6 +1,36 @@
-from typing import List
+import os
 import subprocess
-from pylib.file.file_utils import FileUtils
+from typing import List, Optional
+
+
+def find_src_root() -> str:
+    '''Return the project source root directory.'''
+    dir = os.getcwd()
+    while (
+        dir
+        and dir != '/'
+        and os.path.isdir(dir)
+        and not os.path.exists(os.path.join(dir, '.git'))
+    ):
+        dir = os.path.dirname(dir)
+    return dir
+
+
+def get_absolute_filepath(filename: str) -> Optional[str]:
+    """Return the absolute path for the filename, if it exists."""
+    if not filename:
+        return None
+
+    if os.path.exists(filename):
+        return os.path.normpath(os.path.abspath(filename))
+
+    if filename[0] == '/':
```

Comment:
If the file doesn't exist - shouldn't we be throwing an exception instead? How are we finding the absolute path of a non-existent file?

Reply:
This is same signature that the previously imported utility used (ie returning None when the file doesn't exist) so my preference is to keep it as-is for the sake of consistency and restricting the scope of this PR. 
Two other notes
- Isn't the 'file doesn't exist' case essentially covered?
- I don't actually like the way that `get_absolute_filepath` is used so it's likely that I'll remove it altogether in the future